doc/langdef.md

```diff
@@ -28,7 +28,7 @@ Expr           = ConditionalOr ["?" ConditionalOr ":" Expr] ;
 ConditionalOr  = [ConditionalOr "||"] ConditionalAnd ;
 ConditionalAnd = [ConditionalAnd "&&"] Relation ;
 Relation       = [Relation Relop] Addition ;
-Relop          = "<" | "<=" | ">=" | ">" | "==" | "!=" | "in" ;
+Relop          = "<" | "<=" | ">=" | ">" | "==" | "!=" | " in " ;
```

Comment:
The grammar is whitespace agnostic in practice, so this is a no-op.

Reply:
ok